### ISSUE TITLE
Does not work when headers contain hyphens.

### ISSUE THREAD

**Issue by rossdeane** [keredson/boddle#3]
Headers that contain hyphens are not passed through properly because bottle transforms hyphens into underscores. The following test case fails:
```py
  def testHeaders(self):
    with boddle(headers={'x-header':'value'}):
      self.assertEqual(bottle.request.headers['X-HEADER'], 'value')
```

**Comment by keredson** [comment]
that is weird.  so weird.  learn something new every day:
https://stackoverflow.com/questions/22856136/why-underscores-are-forbidden-in-http-header-names

also weird that bottle doesn't support turning this on and off like nginx/etc.  fully hard-coded:
https://bottlepy.org/docs/dev/_modules/bottle.html#WSGIHeaderDict

nice catch!

**Comment by keredson** [comment]
released in boddle-0.2.6

**Comment by rossdeane** [comment]
Yeah it's strange! Thanks for the merge! 👍
